liquid-rust-tests/tests/neg/user_qualifiers/not_well_formed.rs

```diff
@@ -0,0 +1,11 @@
+#![feature(register_tool)]
+#![register_tool(lr)]
+#![feature(custom_inner_attributes)]
+
+#![lr::qualifier(MyQ (x: int) -> x == a)] //~ ERROR cannot find value `a` in this scope
+#![lr::qualifier(MyQ (x: int) -> x + 1)] //~ ERROR mismatched sorts
```

Comment:
I kind of hate these spaces 😛 

```suggestion
#![lr::qualifier(MyQ(x: int) -> x == a)] //~ ERROR cannot find value `a` in this scope
#![lr::qualifier(MyQ(x: int) -> x + 1)] //~ ERROR mismatched sorts
```